### ISSUE TITLE
Test design against WCAG

### ISSUE THREAD

**Issue by matthiasrohmer** [jungvonmatt/amp.dev#53]
## Objective

The current state of the design should be matched against the [WCAG](https://www.w3.org/TR/WCAG21/). Specifically in terms of color contrast.

**Comment by matthiasrohmer** [comment]
The deliverable of this issue should be a list of things that need to be adjusted in the current design.

**Comment by LongoOfficial** [comment]
Testing design against WCAG means in our team understanding the color contrast ratio of text color and background color. There are other factors to consider in order to be fully compliant with AA standard, but this should be the box we want to tick at least.

With our longer form articles and content texts we even hit AAA both on our white (ratio: 7.96) and light grey backgrounds (ratio: 7.64).

When trying to make our vivid gradients fully WCAG 2.0 (AA) compliant, we would lose a lot of vitality and openness in our visual language.

These were our initial gradients:
<img width="1227" alt="bildschirmfoto 2018-09-12 um 10 51 21" src="https://user-images.githubusercontent.com/41894292/45413589-db385400-b679-11e8-8956-eb8e6a3236d7.png">

There are bright and vivid color transformations within the color spectrum.
But white text misses in some cases a high enough contrast ratio to hit AA.

When making these gradients AA compliant we would end up with these gradients:
<img width="1204" alt="bildschirmfoto 2018-09-12 um 10 56 04" src="https://user-images.githubusercontent.com/41894292/45413866-76c9c480-b67a-11e8-8efd-767272f13593.png">

The visual appearance would change its direction to a more darkish look and feel which is contrary to visual idea of AMP of being inviting and open.


The idea now is to have a gradient were the majority of colors within that gradient work with the required contrast ratio, but also have a highlight top end with bright and vivid colors. We will need to make sure that text is in every case positioned on AA compliant color parts.

<img width="586" alt="bildschirmfoto 2018-09-12 um 11 12 12" src="https://user-images.githubusercontent.com/41894292/45414932-be515000-b67c-11e8-8f84-ff2c5d29607e.png">

This is something we would further explore in the next sprint together with the destination color principles and stage design.

**Comment by pbakaus** [comment]
@spacedino curious to hear your thoughts. Could work, but might be tricky in terms of responsive design.

**Comment by spacedino** [comment]
i know this issue is closed - is there a follow up issue? also how can i filter for all of the UX and visual design tickets?

**Comment by andmckvr13** [comment]
@spacedino, I'm unsure of any follow up issues here, but want to help with filtering issues.   To filter for design issues click labels on the open issues page and select the design labels.

**Comment by keevcodes** [comment]
@spacedino currently all UX design issues are under  the design label. @matthiasrohmer @sebil perhaps we should add a UX label to make it easier to filter for these issues?

**Comment by spacedino** [comment]
yea it's hard for me to review work because i can't tell which issues have actual assets in the issue vs just a "to do". For example, this one : https://github.com/jungvonmatt/amp.dev/issues/154 is a design task but there are no designs in it....  : D

**Comment by matthiasrohmer** [comment]
> i know this issue is closed - is there a follow up issue? also how can i filter for all of the UX and visual design tickets?

As @andmckvr13 pointed out all UX and design issues share the label [department:design](https://github.com/jungvonmatt/amp.dev/issues?q=is%3Aissue+is%3Aopen+label%3Adepartment%3Adesign) or [department:concept](https://github.com/jungvonmatt/amp.dev/issues?q=is%3Aissue+is%3Aopen+label%3Adepartment%3Aconcept) respectively.

If you have any additions to this topic @spacedino, feel free to create a follow up issue 🙂What would help is if you could mention me in the newly created issue - then I can take care of rubbing it in Longo's and Lenny's faces if they are not responding 😉

**Comment by matthiasrohmer** [comment]
> yea it's hard for me to review work because i can't tell which issues have actual assets in the issue vs just a "to do". For example, this one : #154 is a design task but there are no designs in it.... : D

For sure 😄 @LongoOfficial could you please take care of providing the created designs to the finished tickets and mention Abby?

**Comment by LongoOfficial** [comment]
@spacedino I apologize for my bad tracking. It's still a bit unfamiliar for me to await feedback on this channel, although we all agreed on this. Sorry again 🙏.
I'll clean things up and put things in context once I got back to my working machine. I'll keep you posted.
